.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -26,6 +26,8 @@
 
 - [ ] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `<ADAPTER_PACKAGE>/schemas/env.json` and `<ADAPTER_PACKAGE>/README.md`
 - [ ] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `infra-k8s` configuration file.
+- [ ] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant soak testing configuration file or update the [soak testing blacklist](/packages/scripts/src/get-changed-adapters/soakTestBlacklist.ts).
```

Comment:
What would be the "relevant soak testing configuration file"? Is it the test payloads file?

Reply:
I've clarified it slightly now, but the most information about this is written down in internal documentation.